src/components/demo-step-0.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react'
+import { render } from 'react-dom'
+
+import contractService from '../services/contract-service'
+import ipfsService from '../services/ipfs-service'
+import bs58 from 'bs58'
+
+class DemoStep0 extends Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      listingsResults: [],
+      contractListingsCount: -1
+    }
+
+    let that = this
+
+    // Test getting listings from chain
+    setTimeout(function() {
+      // TODO: Remove hacky 2s delay and correctly determine when contractService
```

Comment:
Yea this is fine for now. 

Borrowing from Goldman's modification of the standard TODO, but can we put TODO (name of engineer): .... so that in the future it's really easy for any given engineer to search for their own TODOs.